django/http/request.py

```diff
@@ -61,6 +61,8 @@ def __init__(self):
         self.method = None
         self.resolver_match = None
         self._post_parse_error = False
+        self.content_type = None
+        self.content_params = None
```

Comment:
Do you think `None` is better than an empty string / empty dictionary, respectively? A few other edits: https://dpaste.de/G9P3